src/config/raw.rs

```diff
@@ -462,33 +462,35 @@ mod test {
     #[test]
     #[cfg(all(feature = "yaml_format", feature = "threshold_filter"))]
     fn full_deserialize() {
+        // editorconfig-checker-disable
```

Comment:
did this work? Also idk if we need this, seems like it was working as intended right?

Reply:
Yup works, refer to reason above to keep this.

Reply:
FYI these are still needed. Added comment to explain